astronomer-certified-setup.py

```diff
@@ -53,29 +53,45 @@ def get_info_from_airflow_wheel(filepath):
     return extras, airflow_version
 
 
-def transform_airflow_version(airflow_ver):
+def transform_airflow_version(airflow_ver, build_latest_release=False):
 
     parsed = Version(airflow_ver)
 
     ver = '.'.join(str(x) for x in parsed.release)
 
-    if parsed.local:
-        local = parsed._version.local[1]
-        ver += '-' + str(local)
+    if not build_latest_release:
```

Comment:
As a general rule: I find the less indented code is, the easier it is to read:

```python
    if build_latest_release:
        return airflow_ver

    parsed = Version(airflow_ver)

    ver = '.'.join(str(x) for x in parsed.release) 

    ...
```

Reply:
Updated